src/js/components/ShareButtons.js

```diff
@@ -199,9 +199,21 @@ class ShareButtons extends React.PureComponent {
     }
 
     return (
-      <div id="share-menu">
-        <ul className="share-buttons">{media.map(item => mediaMap[item])}</ul>
-      </div>
+      <>
+        <div id="share-menu">
+          <ul className="share-buttons">{media.map(item => mediaMap[item])}</ul>
+        </div>
+        <div id="fullscreen-share-menu">
```

Comment:
Why has this been made part of ShareButtons component? It doesn't seem to belong here.